packages/taquito-michel-codec/src/michelson-typecheck.ts

```diff
@@ -424,7 +424,7 @@ function assertDataValidInternal(d: MichelsonData, t: MichelsonType, ctx: Contex
       if (
         'string' in d &&
         checkDecodeTezosID(d.string, 'ED25519PublicKey', 'SECP256K1PublicKey', 'P256PublicKey') !==
-        null
+          null
```

Comment:
extra indent not needed

Reply:
this was from the linter :O

Reply:
because its related to the above check `'SECP256K1PublicKey', 'P256PublicKey') !== null`